lib/src/core/verifier/mod.rs

```diff
@@ -0,0 +1,76 @@
+use crate::*;
+use anyhow::Result;
+use async_trait::async_trait;
+use serde_json::{json, Value};
+use std::path::PathBuf;
+
+pub mod policy;
+mod sample;
+mod sgx;
+mod tdx;
+
+#[derive(Debug)]
+pub struct Evidence {
+    pub quote: String,
+    pub ehd: String,
+    pub add: String,
```

Comment:
I think the RFC Issue has `aad` for this field rather than `add`. I assume this refers to `Additional Authenticated Data`?